### ISSUE TITLE
Namespace prefix payload error

### ISSUE THREAD

**Issue by miroslavKovacPantheon** [CiscoDevNet/ydk-gen#849]
When I am using ydk to create data with payload as displayed bellow it will fail with

> ydk.errors.YModelError:  Failed to resolve identityref "arap". Path: /ietf-interfaces:interfaces/interface[name='my_interface']/type

the problematic part is the type of the module which is imported from another module called ietf-iana-type. If I use iana-if-type -which is a module name - instead of the prefix ianaift so the type will look like
 `<type xmlns:ianaift="urn:ietf:params:xml:ns:yang:iana-if-type">iana-if-type:arap</type>`
it will create data just fine.
and when I read them back I will receive data in xml back with the prefix instead.

so the conclusion is that reading and encoding to xml works fine but when I do create with the prefix it will fail

```
from argparse import ArgumentParser

from ydk.services import NetconfService, CodecService, Datastore, CRUDService
from ydk.providers import NetconfServiceProvider, CodecServiceProvider
import logging

payload = '''
  <if:interfaces  xmlns:if="urn:ietf:params:xml:ns:yang:ietf-interfaces">
  <if:interface>
    <name>my_interface</name>
    <type xmlns:ianaift="urn:ietf:params:xml:ns:yang:iana-if-type">ianaift:arap</type>
  </if:interface>
</if:interfaces>
'''

payload2 = '''
  <if:interfaces  xmlns:if="urn:ietf:params:xml:ns:yang:ietf-interfaces">
  
</if:interfaces>
'''

if __name__ == "__main__":
    """Execute main program."""
    parser = ArgumentParser()
    parser.add_argument("-v", "--verbose", help="print debugging messages",
                        action="store_true")
    args = parser.parse_args()

    # log debug messages if verbose argument specified
    if args.verbose:
        logger = logging.getLogger("ydk")
        logger.setLevel(logging.INFO)
        handler = logging.StreamHandler()
        formatter = logging.Formatter(("%(asctime)s - %(name)s - "
                                       "%(levelname)s - %(message)s"))
        handler.setFormatter(formatter)
        logger.addHandler(handler)

    # create NETCONF provider
    nc_provider = NetconfServiceProvider(address='localhost',
                                         port=2022,
                                         username='admin',
                                         password='**********',
                                         protocol='ssh')
    # create Codec provider and service
    xml_cd_provider = CodecServiceProvider(type="xml")
    codec = CodecService()

    # create CRUD service
    crud = CRUDService()

    # decode XML, validate data and create config using NETCONF
    entity = codec.decode(xml_cd_provider, payload, True)
    crud.create(nc_provider, entity)

    entity = codec.decode(xml_cd_provider, payload2, True)
    result = crud.read(nc_provider, entity)
    xml = codec.encode(xml_cd_provider, result )

    exit()
# End of script
```

**Comment by abhikeshav** [comment]
Related to https://github.com/CESNET/libyang/issues/703

**Comment by ygorelik** [comment]
This issue was resolved in YDK-0.8.1
